pkg/report/handler.go

```diff
@@ -28,6 +33,15 @@ func ShowHandler() func(w http.ResponseWriter, r *http.Request) {
 		panic("bucket name was not defined or empty")
 	}
 
+	tmpDir := fmt.Sprintf("%s/kangal", strings.TrimRight(os.TempDir(), "/"))
+
+	go func() {
+		ticker := time.NewTicker(5 * time.Hour)
+		for range ticker.C {
+			os.RemoveAll(tmpDir)
+		}
+	}()
```

Comment:
This is to ensure the local cache is always fresh. This way is not necessary a container restart to cleanup.